packages/vm/src/evm/types.ts

```diff
@@ -19,3 +20,7 @@ export type AddOpcode = {
 }
 
 export type CustomOpcode = AddOpcode | DeleteOpcode
+export interface TxContext {
```

Comment:
i think best to add a newline above so it isn't thought to be related to CustomOpcode above it. even better to have a typedoc for it.

```suggestion

/**
 * TxContext for vm execution
 */
export interface TxContext {
```